test/recursive-raddir-test.js

```diff
@@ -72,4 +72,20 @@ describe('readdir', function() {
       done()
     })
   })
+
+  it('works when negated ignore list is given', function(done) {
+
+    var expectedFiles = [
+      __dirname + '/testdir/c.txt',
```

Comment:
Oh wow, didn't notice the old tests were like that. Don't worry about that,
I'll clean up the old tests myself.

On Wed, Apr 15, 2015 at 9:35 AM, Ferhat Elmas <notifications@github.com>
wrote:

> In test/recursive-raddir-test.js
> <https://github.com/jergason/recursive-readdir/pull/20#discussion_r28431367>
> :
>
> > @@ -72,4 +72,20 @@ describe('readdir', function() {
> >        done()
> >      })
> >    })
> > +
> > +  it('works when negated ignore list is given', function(done) {
> > +
> > +    var expectedFiles = [
> > +      __dirname + '/testdir/c.txt',
>
> sure.
>
> do you want me to edit old test, too?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/jergason/recursive-readdir/pull/20/files#r28431367>.
>